doc/developers/testsuite.html

```diff
@@ -728,6 +728,18 @@
     </p>
 
     <p>
+      It is possible to submit test results for an already installed library by
```

Comment:
then it will read `It is possible to test and submit test results...` with two `test` very close to each other.

Above it also reads `To submit test results to our...`.